### DISCUSSION TITLE
Flynt Intro Video

### DISCUSSION THREAD

**Discussion by aaronmeder** [flyntwp/flynt#482] [General]
Hey Flynt Community!

I made a little video with a quick overview of Flynt:
**Flynt – A Component Based WordPress Starter Theme for Developers To Create Tailored Websites**
https://youtu.be/8as8rVNcJuU

![Flynt1-YoutubeCover-2](https://user-images.githubusercontent.com/5564163/223245776-992661ed-b723-4963-b6e3-5949e6c9f16d.png)

Hope you like it! Gotta say it was harder to do in English than I thought - but overall I am still happy 😅
Would like your suggestions on improvements, should I ever decide do another video again ;-)
Think a more hands-on "build a new component from start to end" video would most likely be next.

Cheers
Aaron

**Comment by domtra** [comment]
Hey @aaronmeder , this is awesome!!!
I really enjoyed watch this video. Great job! You cover a lot of important aspects and show all parts that make up a component. I also really like the switching between slides and screencast. Must have been quite some work.

**Reply by aaronmeder** [reply]
Cool that you liked it @domtra. Took a while to put it all together yep, but was a good experience overall!

**Comment by steffenbew** [comment]
That's great, @aaronmeder! Congrats and thanks for your support!
I think the video provides a very good overview. The length is just right for me and switching between slides, code and demos made it fun to watch.

**Reply by aaronmeder** [reply]
Happy you've liked it too @steffenbew! Hope to shine some more light on Flynt in the future.

**Comment by CodeConvergence** [comment]
Very nicely done Aaron! 🤩
Looking forward to seeing more :)
